### DISCUSSION TITLE
Is there an official method to handle a double press?

### DISCUSSION THREAD

**Discussion by krispya** [adobe/react-spectrum#1585] [Q&A]
Hello, I was wondering if there is an official method for handling a double press event? I can make one easy enough using onPressChange but no reason to do that if it already exists!

**Comment by devongovett** [comment]
There is not! One question I would have would be how to trigger a double click with a screen reader. Perhaps using the dblclick event in addition to any custom mouse/touch handling would be necessary. We would need to test this across screen readers I think.

What is your use case for double clicking?

**Reply by krispya** [reply]
Ah! I hadn't thought about the screen reader scenario. Our project is using React Aria to build out the base functionality for an editor app. We were using double click to initiate renaming a folder (like a typical file tree). This is a shortcut for us because there is also an options menu for single click options, so we see it as an enhancement. I'm not sure if that fits into the React Aria goals.
